metrics/naming.go

```diff
@@ -1,12 +1,12 @@
 package metrics
 
-const appName = "pulse"
+const appName = "block-header-service"
 
 const requestMetricBaseName = "http_request"
 const requestCounterName = requestMetricBaseName + "_total"
 const requestDurationSecName = requestMetricBaseName + "_duration_seconds"
 
-const domainPrefix = "bux_"
+const domainPrefix = "bhs_"
```

Comment:
Worth to discuss if we don't want it consistent across SPV Wallet stack
Maybe `bsv_` would be good prefix ❓ 
WDYT @chris-4chain ?